pkg/config/ebpf_tracer.go

```diff
@@ -55,4 +55,7 @@ type EBPFTracer struct {
 
 	// Enables GPU instrumentation for CUDA kernel launches and allocations
 	InstrumentGPU bool `yaml:"instrument_gpu" env:"BEYLA_INSTRUMENT_GPU"`
+
+	// Enables GPU instrumentation for CUDA kernel launches and allocations
```

Comment:
This comment is a copy paste from `InstrumentGPU`. We will need a follow up PR to fix it

Reply:
Ah nice catch, yeah I'll fix it in a subsequent change. Thanks.